From 70343de1bd8337f8b232528abb6175004c7cf098 Mon Sep 17 00:00:00 2001
From: Michael Wright <michaelwr@google.com>
Date: Wed, 4 Dec 2013 15:23:41 -0800
Subject: [PATCH 313/634] android: configs: Add HIDRAW to recommended set

The Logitech unifying driver depends on hidraw being available.
Recommending one without the other will cause the Logitech driver to
silently fail when connecting Logitech devices.

Change-Id: I92ed2b6803537d9da6eed7fcada8f329cb4469a2
Signed-off-by: Michael Wright <michaelwr@google.com>
---
 android/configs/android-recommended.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android/configs/android-recommended.cfg b/android/configs/android-recommended.cfg
index 1bd9f6f..32596bc 100644
--- a/android/configs/android-recommended.cfg
+++ b/android/configs/android-recommended.cfg
@@ -18,6 +18,7 @@ CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_SECURITY=y
 CONFIG_FUSE_FS=y
 CONFIG_GREENASIA_FF=y
+CONFIG_HIDRAW=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
-- 
1.9.1

